b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/editUserComponent.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from "react";
+import { Field, Form } from "react-final-form";
+import { Button, ButtonToolbar, Loader, Modal, useToaster } from "rsuite";
+import FormSuite from "rsuite/Form";
+import stylesSettings from "../../../../../styles/Settings.module.css";
+import styles from "../../../../../styles/util.module.css";
+import decodeEditUser from "../../../../../util/apiDecode/settings/decodeEditUser";
+import { checkIfJSONisEmpty } from "../../../../../util/util/common/common";
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from "../../../../../util/util/frontendUtil/frontendUtil";
+import { errorTypeDialog, successTypeDialog } from "../../../../common/dialog";
+
+/**
+ * 
+ * @param prop - session, user (user details), open (whether the modal open or close), onClose (on modal close)
+ * @returns Modal form to edit the user
+ */
+export default function EditUserComponent(prop) {
+
+    const { session, user, open, onClose } = prop;
+
+    const toaster = useToaster();
+
+    const [ loadingDisplay, setLoadingDisplay ] = useState(LOADING_DISPLAY_NONE);
+
+    const firstNameValidate = (firstName, errors) => {
+        if (!firstName) {
+            errors.firstName = "This field cannot be empty";
+        }
+
+        return errors;
+    };
+
+    const familyNameValidate = (familyName, errors) => {
+        if (!familyName) {
```

Comment:
Shall we use one common function to validate these fields other that implementing method for each validation?

Reply:
This is fixed with a later PR,

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/shared/util/util-front-end-util/src/lib/formUtil/formUtil.ts 
- https://github.com/wso2/samples-is/blob/09d8f7d6abe21d199f2c739e578b0883825d8cf2/b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/manageUserSection/otherComponents/editUserComponent.tsx#L110

### Commit
- https://github.com/wso2/samples-is/commit/9c330e205151b20f9436bf6aad9c4db85e216673#diff-2982d16cf307c73b8cbce49859f3040b0fa42f10c49acaa3952642cdd40632b5